### PR TITLE
Create nice-code.md

### PR DESCRIPTION
**Pull request by randompea**
# Create the file nice-code.md

My pull request is to
- [x] Add documentation
- [ ] Fix documentation

If adding documenation, I am adding it to
- [ ] Code blocks
- [ ] Objects
- [x] Guides

If adding documentation, did I
- [x] Correctly format the page to match the other ones
- [x] Add it to the index page for the correct category
- [x] Add it to SUMMARY.md

# Additional information

This teaches newbies how to make their code look nice for other newbies.
--- Describe your PR here.

### CODE REVIEW

File: guides/simple/nice-code.md
Comment (markverb1) [inline guides/simple/nice-code.md:1]: change the intro please. maybe make it something like "Have you looked at your code but can't read it?" and also change "How to clean it?" to "How to fix it?"

Comment (markverb1) [reply guides/simple/nice-code.md:1]: change "But a new player still doesn't know what this code means." to "But a new player still wouldn't know what this code means." and as I said in the Discord change all "lets" to "let's"

Comment (klafl6401) [reply guides/simple/nice-code.md:1]: mark -> 🤓

Comment (randompea) [reply guides/simple/nice-code.md:1]: done - thanks for the improvement ideas

**Comment by randompea** [comment]
useless

**Pull request closed without merging**